### DISCUSSION TITLE
CDN 节点地址需要修改！！

### DISCUSSION THREAD

**Discussion by imPrk0** [travellings-link/travellings#1009] [寻求帮助] [answered]
亲爱的**Travelling**，您好：

前段时间，中共政府对 `cdn.jsdelivr.net` 域名进行了污染拦截（被墙）。届时，需要将域名更换为：

``` url
fastly.jsdelivr.net
```
但因 GitHub 上存在大量反共资源，都可以被 jsDelivr 的 CDN 进行回源，所以过不了多久也是会被墙的。包括 GitHub 都在 2020 年两会开始被间歇性墙。所以尽量更换图床才是最佳选择！推荐使用腾讯云智服的图床，速度快不压缩。链接（QQ2或腾讯云智服）：[Prk在线工具网](https://tool.cprk.cc)。

谢谢！

[Prk](https://imprk.me)
2022-05-26

**Comment by linlinzzo** [comment]
站长可以自选存储，我们不会推荐站长使用某个服务。

**Comment by imjason** [comment, marked as answer]
目前已经修改为fastly节点，后续会改到其他更优秀节点
